app/graph/types/query_type.rb

```diff
@@ -83,8 +83,6 @@
 
   instance_exec ProjectMedia, :project_media, ProjectMediaType , &GraphqlCrudOperations.project_association
```

Comment:
if `project_association` is used only by `ProjectMedia` now, we can define it here for clarity

Reply:
yes used for ProjectMedia only and will change it